### DISCUSSION TITLE
Macbook pro without GPU

### DISCUSSION THREAD

**Discussion by EXSYM** [upscayl/upscayl#677] [Q&A]
Hello,

Just wanted to make sure if the app could work in macbook pro without GPU (intel i5 iGPU)

If it's not, is there any way I can use the app on mac using iGPU ?

**Comment by Wyrdgirn** [comment]
Installing Windows or Linux can be used without GPU, with MacOS I think it still cannot be used without GPU...

**Comment by aaronliu0130** [comment]
Try to find .bundle, .so, .dylib or .framework files under the .app contents with the same name as the guide

**Comment by NayamAmarshe** [comment]
Upscayl will soon start to support CPUs (hopefully).
